edxadapt/edxadapt.py

```diff
@@ -55,12 +56,24 @@ class EdxAdaptXBlock(StudioEditableXBlockMixin, XBlock):
         help=_('Edx Adapt API base URL, e.g. https://edx-adapt.example.com:443/api/v1')
     )
 
+    edx_adapt_course_id = String(
+        default='',
+        display_name=_('Edx Adapt CourseID'),
+        scope=Scope.content,
+        help=_(
+            "By default CourseID is taken from the Edx e.g. 'course-v1:<University>+<Course_name>+<course_run>'."
```

Comment:
So is `course-v1` a part of course id or not? This line and 67 sort of contradict with each other.